lib/gamnit/bmfont.nit

```diff
@@ -406,8 +415,58 @@ class BMFontAsset
 					else 16.0
 				dx += space_advance * scale
 				word_break = true
+			else if c == '[' then
+				# Open link?
+				if i + 1 < text.length and text[i+1] == '[' then
+					# Escape if duplicated
+					i += 1
+				else
+					in_link = true
+					continue
+				end
+			else if c == ']' then
+				# Close link?
+				if i + 1 < text.length and text[i+1] == ']' then
+					# Escape if duplicated
+					i += 1
+				else
+					# If there's a () use it as link_name
+					var j = i + 1
+					if j < text.length and text[j] == '(' then
+						var new_name: nullable String
```

Comment:
Hum... right but it's weird:

This works
~~~
var new_name
new_name = ""
~~~

But this alternative raises an error when assigning `new_name = null` later:
~~~
var new_name = ""
~~~

While this raise a "method call on nullable" further down:
~~~
var new_name: nullable String = ""
~~~